test/dynamo/test_repros.py

```diff
@@ -872,7 +872,7 @@ def test_reformer_train(self):
         with torch.enable_grad():
             cnt = self._reformer(nopython=False)
         # cant inline torch.autograd.Function means graph break
-        self.assertEqual(cnt.frame_count, 4)
+        self.assertEqual(cnt.frame_count, 3)
```

Comment:
No graph break on ```ctx.save_for_backward```, so save one frame count. Actually this unit test triggers re-complication before this PR, but we didn't catch it.

Reply:
Yeah I've noticed frame count is really not the best way of detecting these. I started using explain() to count the graphs and graph breaks